src/app/components/Application/Application.jsx

```diff
@@ -72,6 +73,10 @@ class App extends React.Component {
     return (
       <DocumentTitle title="Shared Collection Catalog | NYPL">
         <div className="app-wrapper">
+          <DataLoader
+            key={JSON.stringify(this.context.router.location)}
```

Comment:
You don't need the `key` attribute here.

Reply:
This key is just to re-trigger componentDidMount when the path changes. Could also use componentDidUpdate, but that seems to be trickier since then properties persist from one load to the next.